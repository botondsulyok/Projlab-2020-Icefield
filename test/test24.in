field spawn 0
entity master Eskimo spawn
item pist1 Pistol master
item cart1 Cartridge master
item light1 Light master
item pist2 Pistol master
item cart2 Cartridge master
item light2 Light master
stats spawn
stats master
assemble master
stats spawn
stats master