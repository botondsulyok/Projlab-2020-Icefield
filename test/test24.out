id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master
pullers: master
masters: master
neighbours: null

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1, cart1, light1, pist2, cart2, light2

Game Won!
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master
pullers: master
masters: master
neighbours: null

id: master
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1, cart1, light1, pist2, cart2, light2

